### DISCUSSION TITLE
How Part Payment can be made on BAP

### DISCUSSION THREAD

**Discussion by ankitagarwaladobe** [ONDC-Official/ONDC-FIS-Specifications#10] [Q&A]
Is it possible to make a part paymet from a BAP?

**Comment by Antriksh-Parmar** [comment]
BAP can request the lender to share a link for part payments. @92shreyansh pls help with the API that enables this use-case.

**Comment by 92shreyansh** [comment]
The current version of draft has no explicite examples for this use case will add it in a day or two. 

Fundamentally the update and on_update api would handle this where in the BAP would request the BPP for a pre-payment with a identified amount from the buyer in a update API and the BPP would respond back with an on_update with a payment link.

**Reply by ankitagarwaladobe** [reply]
So there would be an update call to initiate a part payment and get a link.

Once the borrower has made the payment, then the same would be reflected using a status, on_status apis?
